indy_node/test/auth_rule/test_constraint_field.py

```diff
@@ -131,3 +131,25 @@ def test_auth_constraint_from_dct_succesfull():
         if k != unkwn:
             assert hasattr(constr, k)
     assert not hasattr(constr, unkwn)
+
+
+def test_auth_constraint_without_off_ledger_sig_from_dct_succesfull():
+    unkwn = 'unknown_field'
+    auth_dct = {CONSTRAINT_ID: ConstraintsEnum.ROLE_CONSTRAINT_ID,
+                ROLE: STEWARD,
+                SIG_COUNT: 1,
+                NEED_TO_BE_OWNER: False,
+                METADATA: {},
+                unkwn: 'unknown_value'}
+    constr = AuthConstraint.from_dict(auth_dct)
+
+    for k in auth_dct:
+        if k != unkwn:
+            assert hasattr(constr, k)
+    assert not hasattr(constr, unkwn)
+    assert constr.off_ledger_signature is None
+
+    del auth_dct[unkwn]
+    print(auth_dct)
+    print(constr.as_dict)
```

Comment:
Why prints instead of logger?

Reply:
Changes in https://github.com/hyperledger/indy-node/pull/1418